tidy3d/web/core/http_util.py

```diff
@@ -129,6 +131,10 @@ def wrapper(*args, **kwargs):
         if resp.status_code != ResponseCodes.OK.value:
             if resp.status_code == ResponseCodes.NOT_FOUND.value:
                 return None
+            if resp.status_code == ResponseCodes.UNPROCESSABLE.value and resp.text:
+                result = resp.json()
+                if str(ResponseCodes.TASK_REPEATED.value) == result["code"]:
```

Comment:
what's the reason for `str()` casting here and not in the other `if` statements above?